data/types/device/alarm/alarm_test.go

```diff
@@ -64,6 +77,9 @@ func CloneAlarm(datum *alarm.Alarm) *alarm.Alarm {
 		}
 	}
 	clone.StatusID = pointer.CloneString(datum.StatusID)
+	clone.AlarmLevel = pointer.CloneString(datum.AlarmLevel)
+	clone.AlarmCode = pointer.CloneString(datum.AlarmCode)
+	clone.AlarmLabel = pointer.CloneString(datum.AlarmLabel)
```

Comment:
```suggestion
	clone.AlarmLabel = pointer.CloneString(datum.AlarmLabel)
	clone.AckStatus = pointer.CloneString(datum.AckStatus)
	clone.UpdateTime = pointer.CloneString(datum.UpdateTime)
```
Missing AckStatus & UpdateTime

Reply:
good catch